netlify.toml

```diff
@@ -0,0 +1,5 @@
+# Netlify build and deploy settings - https://docs.netlify.com/configure-builds/file-based-configuration/#sample-file
+[build]
+base = "packages/core"
+command = "yarn build"
```

Comment:
maybe better to do something like `yarn build-storybook` here to prevent unnecessary build steps to sneak in for netlify as they get added to the general build target